PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2778,7 +2778,9 @@ void ScintillaEditView::currentLinesDown() const
 	execute(SCI_SCROLLRANGE, execute(SCI_GETSELECTIONEND), execute(SCI_GETSELECTIONSTART));
 }
 
-void ScintillaEditView::changeCase(__inout wchar_t * const strWToConvert, const int & nbChars, const TextCase & caseToConvert) const
+namespace {
+
+void changeCase(__inout wchar_t * const strWToConvert, int nbChars, TextCase caseToConvert)

```

Comment:
Please restore this part.

Reply:
I reverted that and a few other drive-by changes.